target/jamba.vpy

```diff
@@ -224,4 +224,21 @@ if (flb := rc['flowblur'])['enabled'].lower() in YES and rc['flowblur']['enabled
         verb(clip)
 # flowblur
 
+cg = rc["color grading"]
+
+brightness = float(cg["brightness"])
+contrast = float(cg["contrast"])
+saturation = float(cg["saturation"])
+hue = float(cg["hue"])
```

Comment:
Assumes hue is set in recipe.ini

Reply:
it will be present in defaults.ini either way 👍